gensim/models/atmodel.py

```diff
@@ -19,14 +19,31 @@
 The model is closely related to Latent Dirichlet Allocation. The AuthorTopicModel class
 inherits the LdaModel class, and its usage is thus similar.
 
-Distributed computation and multiprocessing is not implemented at the moment, but may be
-coming in the future.
-
-The model was introduced by Rosen-Zvi and co-authors in 2004
-(https://mimno.infosci.cornell.edu/info6150/readings/398.pdf).
-
-A tutorial can be found at
-https://github.com/RaRe-Technologies/gensim/tree/develop/docs/notebooks/atmodel_tutorial.ipynb.
+The model was introduced by Rosen-Zvi and co-authors in 2004 and is described in 
```

Comment:
Also, it's a good idea to add basic description here
- what's an use cases for this model
- how this works (in general)